src/components/coin-divider/SmallBalancesWrapper.js

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import styled from 'styled-components/primitives';
 import { useOpenSmallBalances } from '../../hooks';
 import { OpacityToggler } from '../animations';
+import { CoinRowHeight } from '../coin-row';
 
-const Content = styled.View.attrs(({ isSmallBalancesOpen }) => ({
+const Content = styled.View.attrs(({ isSmallBalancesOpen, size }) => ({
+  height: size * CoinRowHeight,
   pointerEvents: isSmallBalancesOpen ? 'auto' : 'none',
 }))`
   left: 0;
   margin-top: 13;
   opacity: ${({ isSmallBalancesOpen }) => (isSmallBalancesOpen ? 1 : 0)};
-  position: absolute;
```

Comment:
Just making sure this won't affect iOS. To be safe you could do:
```
${Platform.OS === 'ios' ? 'position: absolute;' : ''}
```

Reply:
just confirmed this doesnt affect iOS 👌️